### PR TITLE
Backend: Learn Page

### PR DESCRIPTION
**Pull request by kave**
@KimberlyMunoz 
@kurtw 

screen shot 2016-01-05 at 10 14 57 am

### CODE REVIEW

File: cfgov/jinja2/v1/learn-page/index.html [hunk 1]
```diff
@@ -43,6 +43,19 @@
 {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
 
 {% block content_main %}
+    {% for block in page.header -%}
+            <div class="block
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/learn-page/index.html]: Extra indentation here

File: cfgov/jinja2/v1/learn-page/index.html [hunk 2]
```diff
@@ -145,5 +146,32 @@
...
+                        {{ render_stream_child(block) }}
+                    </h3>
+                {% elif 'slug' in block.block_type %}
+                    <h2 class="header-slug">
+                        <span class="header-slug_inner">
+                            {{ render_stream_child(block) }}
+                        </span>
+                    </h2>
+                {% elif 'paragraph' in block.block_type %}
+                    <div class="short-desc">
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/learn-page/index.html]: Should this be a `p` tag?

Comment (kave) [inline cfgov/jinja2/v1/learn-page/index.html]: @jimmynotjim nah remember wagtail ignores those tags. non-formatted text in richtext fields like this by default are surrounded by `p` tags. so having a div with the assigned class name creates the same effect.

File: cfgov/jinja2/v1/learn-page/index.html [hunk 3]
```diff
@@ -145,5 +146,32 @@
...
-    }) }}
+        }) }}
+    {% endif %}
+
+    {% for block in page.sidefoot %}
+        <div class="block block__flush-top">
+            {% if 'related_posts' in block.block_type %}
+                {{ related_posts.render(block) }}
+            {% else %}
+                {% if 'heading' in block.block_type %}
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/learn-page/index.html]: Combine `else` and `if` statements.

Comment (kave) [inline cfgov/jinja2/v1/learn-page/index.html]: @anselmbradford donzo

**Pull request merged**